wallet/wallet-cli-commands/src/lib.rs

```diff
@@ -1183,30 +1217,113 @@ pub fn get_repl_command(cold_wallet: bool, mutable_wallet: bool) -> Command {
         repl_command
     };
 
-    // Customize the help template for all commands to make it more REPL friendly
+    lazy_static::lazy_static! {
+        static ref CUSTOM_RPC_TO_CLI_NAME_MAPPINGS: BTreeMap<&'static str, &'static str> =
+            BTreeMap::from([("account_extended_public_key", "account-extended-public-key-as-hex")]);
+    }
```

Comment:
Note: previously this wasn't needed because we allowed one name to be a prefix of the other (due to the usage of `zip` during comparison). Now we require the names to be identical (except for the separator)